# Patch from Andreas Ntaflos <daff@dword.org> to fix compilation issue
# 15 Mar 2007 Marcus D. Hanwell <cryos@gentoo.org> from bug 171004.
--- src/fileimporter.h	2007-03-15 05:55:41.000000000 +0100
+++ src/fileimporter.h.new	2007-03-15 05:56:45.000000000 +0100
@@ -39,7 +39,7 @@
         ~FileImporter();
 
         File* load ( const QString& text );
-        virtual File* load( QIODevice *iodevice ) = NULL;
+        virtual File* load( QIODevice *iodevice ) = 0;
 
         static bool guessCanDecode( const QString & ) { return FALSE; };
 
